src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/test_batch_commands.py

```diff
@@ -574,15 +573,15 @@ def test_batch_get_model_attrs(self):
         attrs = list(self.command_job._get_attrs(models.ResourceFile, 'task.resource_files'))
         self.assertEqual(len(attrs), 3)
         attrs = list(self.command_job._get_attrs(models.JobManagerTask, 'job.job_manager_task'))
-        self.assertEqual(len(attrs), 13)
+        self.assertEqual(len(attrs), 14)
```

Comment:
Are these assertion valid in live test?

Reply:
It should. This just check how many parameters the command should have.

Reply:
Ok. Please fix the CI. Otherwise looks good to me. I can merge as soon as CI pass and @annatisch signs off.

Reply:
Okay, I think I have CI fixed, we'll see after the re-run